src/torchfem/base.py

```diff
@@ -192,7 +192,7 @@ def assemble_stiffness(self, k: Tensor, con: Tensor) -> torch.sparse.Tensor:
 
             K += torch.sparse_coo_tensor(indices, values, size=size).coalesce()
 
-        return K
```

Comment:
Maybe you can `coalesce` the added entity every now and then, unless it is really huge.